tests/playbooks/tests_bond_deprecated.yml

```diff
@@ -3,7 +3,7 @@
 - hosts: all
   vars:
     controller_profile: bond0
-    controller_device: nm-bond
+    controller_device: bond
```

Comment:
Please consider using a less generic interface name to make it more clear where the interface comes from, maybe `deprecated-bond`.

Reply:
Changed.